psalm.xml.dist

```diff
@@ -166,5 +166,12 @@
                 <directory name="vendor/nikic/php-parser" />
             </errorLevel>
         </MixedAssignment>
+
```

Comment:
Could you suppress them with an @psalm-suppress instead please? Putting them here would mean any future usage of those outside of tests would not get flagged by Psalm so I'd like to avoid this

Reply:
done